serverless/serverless
# Rely on graceful-fs for concurrent file operations (lambda packaging)

## What did you implement:

Switched from using `fs` directly to `graceful-fs` in lambda packaging logic.

After #3924 was taken in, high concurrency may occur in file system operations (due to that, users may encounter `EMFILE` or `ENFILE` errors, as e.g. [reported by @pmuens](https://github.com/serverless/serverless/pull/3924#pullrequestreview-58041963))

## How did you implement it:

Replaced require to `fs` to `graceful-fs` in corresponding places

## How can we verify it:

Run tests, and @pmuens please verify it fixes issue you observed in your setup.

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
